### PR TITLE
Added Turkish locale

### PR DESCRIPTION
**Pull request by muratbsts**
I was not able to run the tests. 😞

### CODE REVIEW

**Comment by natebird** [comment]
As for the failing tests - shouldn't we be using the translation keys in the tests instead of the English strings?

**Comment by muratbsts** [comment]
Hi, @natebird @olbrich I fixed the issues that you said. I ran tests on my machine with ruby v2.4.3 + rails v4.2/5.0/5.2

But travis jobs failed again. ☹️ 

Thank you, I learned new things about of testing.

File: lib/generators/devise_security/install_generator.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ module DeviseSecurity
   module Generators
     # Generator for Rails to create or append to a Devise initializer.
     class InstallGenerator < Rails::Generators::Base
-      LOCALES = %w[ en de it ]
+      LOCALES = %w[ en es de it tr ].freeze
```
Comment (olbrich) [inline lib/generators/devise_security/install_generator.rb]: please add `fr` to this list before the `it` and the tests will pass.  Also please remove extra spaces at start and end.

Comment (muratbsts) [reply lib/generators/devise_security/install_generator.rb]: 👍 done

**Pull request merged**